packages/salmon-fetch/__test__/FetchAsJson.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { fetchAsJson } from '../src'
+import nock from 'nock'
+import { BigNumber } from '@defichain/jellyfish-json'
+
+describe('single asset price fetch', () => {
+  afterEach(() => {
+    jest.clearAllMocks()
+    nock.cleanAll()
+  })
+
+  describe('FetchAsJson', () => {
```

Comment:
```suggestion
  describe('FetchAsJson', () => {
    afterEach(() => {
    jest.clearAllMocks()
    nock.cleanAll()
  })
```

afterEach should be nested inside FetchAsJson